EssentialFeed/EssentialFeediOS/ImageComments UI/Controllers/ImageCommentCellController.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Copyright © 2019 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import EssentialFeed
+
+public final class ImageCommentCellController: NSObject {
+	private let viewModel: ImageCommentViewModel
+	private var cell: ImageCommentCell?
+
+	public init(viewModel: ImageCommentViewModel) {
+		self.viewModel = viewModel
+	}
+}
+
+extension ImageCommentCellController: UITableViewDataSource {
+	public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+		1
+	}
+
+	public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		cell = tableView.dequeueReusableCell()
+		cell?.usernameLabel.text = viewModel.username
+		cell?.createAtLabel.text = viewModel.createdAt
+		cell?.messageLabel.text = viewModel.message
+		return cell!
```

Comment:
There's no need to hold a reference to the cell in this controller (see my previous comment).

```suggestion
		let cell: ImageCommentCell = tableView.dequeueReusableCell()
		cell.usernameLabel.text = viewModel.username
		cell.createAtLabel.text = viewModel.createdAt
		cell.messageLabel.text = viewModel.message
		return cell
```